go/bitsequence.go

```diff
@@ -0,0 +1,47 @@
+package bitsequence
+
+import (
+	"math"
+)
+
+func BitSequence(bytes []byte, bitStart uint32, bitLength uint32) uint32 {
+	if bitLength > 32 {
+		// does this constraint deserve an Error?
+		bitLength = 32
+	}
+	startOffset := bitStart % 8
+	byteCount := uint32(math.Ceil((float64(startOffset) + float64(bitLength)) / 8))
```

Comment:
If I'm not mistaken, this should also work:
```suggestion
	byteCount := (7 + startOffset + bitLength) / 8
```

Reply:
yep, that works, thanks!